test/unit_tests/unittest_scanner_reply_msg.cpp

```diff
@@ -34,80 +35,90 @@ static constexpr uint32_t RES_CODE_ACCEPTED{ 0x00 };
 
 TEST(ScannerReplyMsgTest, testTypeStart)
 {
-  ScannerReplyMsg msg(OP_CODE_START, RES_CODE_ACCEPTED);
-  EXPECT_EQ(ScannerReplyMsgType::start, msg.type());
+  scanner_reply::Type type = scanner_reply::Type::start;
+  scanner_reply::OperationResult result = scanner_reply::OperationResult::accepted;
+  scanner_reply::Message msg(type, result);
```

Comment:
Could be simpler with less lines of code (This comments applies for all tests in this test suite!):
```suggestion
  const scanner_reply::Message msg(scanner_reply::Type::start, scanner_reply::OperationResult::accepted);
```

Reply:
Done.